internal/msgs/en_config_descriptions.go

```diff
@@ -27,17 +27,16 @@ var ffc = func(key, translation string, fieldType string) i18n.ConfigMessageKey
-	ConfigServerWriteTimeout = ffc("config.connectors[].server.writeTimeout", "The maximum time to wait when writing to a HTTP connection", "duration")
-	ConfigAPIShutdownTimeout = ffc("config.connectors[].server.shutdownTimeout", "The maximum amount of time to wait for any open HTTP requests to finish before shutting down the HTTP server", i18n.TimeDurationType)
-
-	ConfigEthereumURL                 = ffc("config.connectors[].ethereum.url", "URL of JSON/RPC endpoint for the Ethereum node/gateway", "string")
-	ConfigEthereumProxyURL            = ffc("config.connectors[].ethereum.proxy.url", "Optional HTTP proxy url", "string")
-	ConfigEthereumDataFormat          = ffc("config.connectors[].ethereum.dataFormat", "Configure the JSON data format for query output and events", "map,flat_array,self_describing")
-	ConfigEthereumGasEstimationFactor = ffc("config.connectors[].ethereum.gasEstimationFactor", "The factor to apply to the gas estimation to determine the gas limit", "float")
+	ConfigEthereumURL                 = ffc("config.connector.url", "URL of JSON/RPC endpoint for the Ethereum node/gateway", "string")
+	ConfigEthereumProxyURL            = ffc("config.connector.proxy.url", "Optional HTTP proxy url", "string")
+	ConfigEthereumDataFormat          = ffc("config.connector.dataFormat", "Configure the JSON data format for query output and events", "map,flat_array,self_describing")
+	ConfigEthereumGasEstimationFactor = ffc("config.connector.gasEstimationFactor", "The factor to apply to the gas estimation to determine the gas limit", "float")
+	ConfigBlockCacheSize              = ffc("config.connector.blockCacheSize", "Maximum of blocks to hold in the block info cache", i18n.IntType)
+	ConfigBlockCacheTTL               = ffc("config.connector.blockCacheTTL", "Time to live for the block info cache", i18n.TimeDurationType)
+	ConfigBlockPollingInterval        = ffc("config.connector.blockPollingInterval", "Interval for polling to check for new blocks", i18n.TimeDurationType)
+	ConfigEventsBlockTimestamps       = ffc("config.connector.events.blockTimestamps", "Whether to include the block timestamps in the event information", i18n.BooleanType)
+	ConfigEventsCatchupPageSize       = ffc("config.connector.events.catchupPageSize", "Number of blocks to query per poll when catching up to the head of the blockchain", i18n.IntType)
+	ConfigEventsCatchupThreshold      = ffc("config.connector.events.catchupThreshold", "How many blocks behind the chain head an event stream or listener must be on startup, to enter catchup mode", i18n.IntType)
+	ConfigEventsCheckpointBlockGap    = ffc("config.connector.events.checkpointBlockGap", "The number of blocks at the head of the chain that should be considered unstable for re-org, such that a checkpoint for an inactive listener should always be this many blocks behind the chain to avoid missing events on restart (in the case a re-org happens that includes new matching events at the head of the chain)", i18n.IntType)
```

Comment:
I think I understand what this means. But this surely wins the award for longest sentence in a config description.

Reply:
ok - will try again...

Reply:
Better?

> The number of blocks at the head of the chain that should be considered unstable (could be dropped form the canonical chain after a re-org). Unless events with a full set of confirmations are detected, the restart checkpoint will this many blocks behind the chain head.